#!/usr/bin/env python
PACKAGE = "road_layout_estimation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("odom_rate", double_t, 0, "Publishing rate", 1,  1, 75)
gen.add("change_direction", int_t, 0, "Numbers of step before changing direction", 5, 1, 50)
gen.add("position_uncertainty", double_t, 0, "Measurement model uncertainty (m)", .05, 0.001, 2)
gen.add("orientation_uncertainty", double_t, 0, "Measurement model uncertainty (m)", .05, 0.001, 2)
gen.add("linear_uncertainty", double_t, 0, "Measurement model uncertainty (rad)", .05, 0.001, 2)
gen.add("angular_uncertainty", double_t, 0, "Measurement model uncertainty (rad)", .05, 0.001, 2)

exit(gen.generate(PACKAGE, "visual_odometry_test", "visual_odometry_test"))
